output.cpp

```diff
@@ -424,14 +424,14 @@ namespace marxan {
         if (itemp >= 60 * 60)
         {
             printf(" %i hour%c,%i min%c and %i secs \n",
-                itemp / 3600, ((itemp / 3600 == 1) ? ' ' : 's'),
-                (itemp / 60) % 60, ((itemp / 60 == 1) ? ' ' : 's'), itemp % 60);
+                (int)(itemp / 3600), ((itemp / 3600 == 1) ? ' ' : 's'),

```

Comment:
what was the issue with the uint64 type? I foudn that with long running times, int will overflow

Reply:
At output %i  expecting integer. With this conversions it should be good up to 800 million years.